spec/dummy-app/spec/spec_helper.rb

```diff
@@ -44,6 +44,11 @@
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  # Allows RSpec to persist some state between runs in order to support
+  # the `--only-failures` and `--next-failure` CLI options. We recommend
+  # you configure your source control system to ignore this file.
```

Comment:
Can we add this file to our `.gitignore` as rails suggests?

Reply:
This was added to `.gitignore` for both the gem and the `dummy-app`